src/transactions/feed/TransactionDetailsScreen.test.tsx

```diff
@@ -290,4 +321,34 @@ describe('TransactionDetailsScreen', () => {
     const subtotal = getByTestId('TotalLineItem/Subtotal')
     expect(getElementText(subtotal)).toEqual('10.10 cUSD')
   })
+
+  it('renders correctly for cUSD to cEUR swap', async () => {
```

Comment:
i feel like this test is the same as the `SwapFeedItem` test :/ do we need both?

Reply:
This test is for the detail page after you click the swap feed item, not the swap feed in the home screen.